docs/sphinx/examples/1d_stencil.rst

```diff
@@ -143,20 +143,20 @@ assist us with managing our asynchrony. For example, our client class
 
 In the structure ``stepper``, we have also had to make some changes to
 accommodate a distributed environment. In order to get the data from a
-neighboring partition, which could be remote, we must retrieve the data from the
-neighboring partitions. These retrievals are asynchronous and the function
-``heat_part_data``, which amongst other things calls ``heat``, should not be
-called unless the data from the neighboring partitions have arrived. Therefore
+particular neighboring partition, which could be remote, we must retrieve the data from all
+of the neighboring partitions. These retrievals are asynchronous and the function
+``heat_part_data``, which, amongst other things, calls ``heat``, should not be
+called unless the data from the neighboring partitions have arrived. Therefore,
 it should come as no surprise that we synchronize this operation with another
-instance of dataflow (found in ``heat_part``). This dataflow is passed futures
+instance of dataflow (found in ``heat_part``). This dataflow passes futures
```

Comment:
I think the original is accurate, but maybe a bit clunky. It's essentially trying to say: futures to the data in the current and surrounding partitions (which one gets by calling get_data) are passed to dataflow. Futures point to data. That's also a clunky way of saying it but does that make the intention of the sentence clear to you?

Reply:
I'm about to push the correction for this. If the correction still sounds wrong, I'll revert it back to the original since it is technically accurate, if awkward.

Reply:
Thanks, that looks good!